examples/xo_android_client/app/src/main/AndroidManifest.xml

```diff
@@ -22,8 +22,8 @@
             </intent-filter>
         </activity>
         <activity
-                android:name=".GameBoardActivity"
```

Comment:
Are we still using the GameBoardActivity class or should it be removed?

Reply:
Yes, it's currently acting as the activity for the game board ui.